.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+
+### What does this PR do?
+
+<!-- A brief description of the change being made with this pull request. -->
+
+
+### Motivation
+
+<!-- What inspired you to submit this pull request? -->
+
+
+### More
+
+- [ ] Yes, I have tested the PR using my local account setup  (Provide any test evidence report under Additional Notes)
+
+- [ ] Yes, I have added a new example under [examples](https://github.com/aws-samples/crossplane-aws-blueprints/tree/main/examples) to support my PR
+
+- [ ] Yes, I have updated the [docs](https://github.com/aws-samples/crossplane-aws-blueprints/tree/main/doc) for this feature
+
```

Comment:
Add something about linking to issues when applicable. 
```
Yes, I have linked to an issue or feature request (applicable to PRs that solves a bug or a feature request)
```

Reply:
updated